ui/app.js

```diff
@@ -3868,6 +3868,9 @@ $(document).ready(() => {
         $("#reports-officers-involved-tag-title").html(
           "Officers Involved"
         );
+        $("#bolos-officers-involved-tag-tittle").html(

```

Comment:
This is duplicated from above

Reply:
not duplicated
one is incident and the other is bolos